library/spdm_responder_lib/libspdm_rsp_measurement_extension_log.c

```diff
@@ -0,0 +1,149 @@
+/**
+ *  Copyright Notice:
+ *  Copyright 2024 DMTF. All rights reserved.
+ *  License: BSD 3-Clause License. For full text see link: https://github.com/DMTF/libspdm/blob/main/LICENSE.md
+ **/
+#include "internal/libspdm_responder_lib.h"
+
+#if LIBSPDM_ENABLE_CAPABILITY_MEAS_CAP
+
+libspdm_return_t libspdm_get_response_measurement_extension_log(libspdm_context_t *spdm_context,
+                                                                size_t request_size,
+                                                                const void *request,
+                                                                size_t *response_size,
+                                                                void *response)
+{
+    const spdm_get_measurement_extension_log_request_t *spdm_request;
+    spdm_measurement_extension_log_response_t *spdm_response;
+    uint16_t offset;
```

Comment:
Both `offset` and `length` are 32 bits.

Reply:
Thanks. I have fixed the variable data type.